src/main/webapp/app/exercises/modeling/shared/modeling-editor.component.ts

```diff
@@ -48,6 +52,34 @@ export class ModelingEditorComponent extends ModelingComponent implements AfterV
             }
         });
         this.setupInteract();
+
+        // Detect Safari desktop: https://stackoverflow.com/a/52205049/7441850
+        const isSafariDesktop = /Safari/i.test(navigator.userAgent) && /Apple Computer/.test(navigator.vendor) && !/Mobi|Android/i.test(navigator.userAgent);
```

Comment:
```suggestion
        this.setupInteract();
        this.setupSafariScrollFix();
    }

    /**
     * This is a hack to prevent the UI jumping to bottom when using popovers in Apollon while using Safari.
     * Other browsers do not show this behavior.
     */
    private setupSafariScrollFix(): void {
        // Detect Safari desktop: https://stackoverflow.com/a/52205049/7441850
        const isSafariDesktop = /Safari/i.test(navigator.userAgent) && /Apple Computer/.test(navigator.vendor) && !/Mobi|Android/i.test(navigator.userAgent);
```